src/app/ui/auto-complete/select/select.component.ts

```diff
@@ -121,6 +124,10 @@ export class SelectComponent implements AfterViewInit, OnDestroy {
     @Input()
     public labelField = "text";
 
+    // When onChange is triggered and this flag is false we know that change comes during items/options initializing process
+    // and we should not propagate the change (if we would, we would always have form dirty flag set)
+    protected shouldTriggerChange = false;
```

Comment:
We have this problem in several places in the application. I don't know how to feel about having to keep track of flags like this all over the place...